dynamiq/components/embedders/base.py

```diff
@@ -101,6 +109,9 @@ def validate_document_embeddings(documents: Any) -> None:
     truncate: str | None = None
     input_type: str | None = None
     dimensions: int | None = None
+    enable_truncation: bool = True
```

Comment:
we talked a lot about `enable_truncation` vs `truncation_enabled`. It easy project search of `enable` vs `enabled` and use proper. Please be careful